src/components/Figure/Image.js

```diff
@@ -26,29 +53,44 @@ class Image extends Component {
       maxWidth,
       size: sizeProp,
       aboveTheFold,
-      enableGallery = true
+      enableGallery = true,
+      gallerySize
     } = this.props
 
     const onClick = (enableGallery && this.context.toggleGallery) ? this.context.toggleGallery : () => {}
```

Comment:
```suggestion
     const isClickable = (enableGallery && this.context.toggleGallery)
     const onClick = isClickable
      ? () => this.context.toggleGallery(src)
      : () => {}
```
create a function to call `toggleGallery` with `src` here and then directly use `onClick` instead of an additional `handleClick` handler.

Also add a cursor `pointer` or `zoom-in` if `isClickable`.